public.yaml

```diff
@@ -61,6 +61,35 @@ metadata:
         ]
       },
       {
+        "name": "salt-minion-key-generation",
+        "image": "sles12/salt-master:2016.11.4",
+        "command": ["sh", "-c", "umask 377;
+                                 salt-key --gen-keys=ca --gen-keys-dir /tmp;
+                                 salt-key --gen-keys=admin --gen-keys-dir /tmp;
```

Comment:
Yeah - had not considered this - I **think** salt would still work, but it is better to be safe. Will add ^